Sources/UAPusher/UAResponse.swift

```diff
@@ -1,12 +1,18 @@
 import Vapor
 import HTTP
 
+/// Urban Airship response
 final public class UAResponse {
+    
     public let responses: [Response]
     
     public var status: Status {
         for response in responses {
             if response.status != .accepted {
+                responses.forEach { response in
+                    print(response.makeResponse())
```

Comment:
Same comment as in the `send` method

Reply:
Removing for now... Creating an issue for the config-idea

Reply:
actually this was in the initial relase, so leaving it for now